istio/deploy/helm/istio/templates/mesh-logging.telemetry.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: telemetry.istio.io/v1
+kind: Telemetry
+metadata:
+  name: mesh-logging
```

Comment:
nit: @SudoBrendan was advocating to explicitely add the namespace as `
namespace: '{{ .Release.Namespace }}'